recipes/graphthewy/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class GraphthewyConan(ConanFile):
+    name = "graphthewy"
+    version = "1.1"
```

Comment:
```suggestion
```

Version should be solved by command line, e.g.  conan create . graphthewy/1.1@

Reply:
Thank you for the review. PR updated.